docs/index.md

```diff
@@ -13,7 +13,22 @@ There are 3 main components in BtcTransmuter - External Services, Recipes and Ex
 
 
 ## How do I deploy?
-You don't, this is alpha software man!
+
+First, create a new A Record in the DNS Settings of your web domain host, for example `transmuter.example.com`
+The IP should be the same one used for your BTCPay Server.
+
+If you set up BTCPay using any dockerized install method, you can enable BTC Transmuter. First, [SSH](https://github.com/JeffVandrewJr/patron/blob/master/SSH.md) into your server:
+
+```bash
+sudo su -
+cd btcpayserver-docker
+export BTCPAYGEN_ADDITIONAL_FRAGMENTS="opt-add-btctransmuter"
```

Comment:
If they have pruning enabled, running only `export BTCPAYGEN_ADDITIONAL_FRAGMENTS="opt-add-btctransmuter"` will remove it and any other env vars they have. They should first retrieve their current env vars and then add transmuter to that list.
```
# retrieve your environment variables
echo $BTCPAYGEN_ADDITIONAL_FRAGMENTS

# add btcTransmuter to your environment variables list that is shown (example below: pruning, btcTransmuter)
export BTCPAYGEN_ADDITIONAL_FRAGMENTS="opt-save-storage-s;opt-add-btctransmuter"
```
Unless there is a better add command? @Kukks do you know?